packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewContent.js

```diff
@@ -0,0 +1,60 @@
+import { LightningElement, api } from 'lwc';
+// @ts-ignore
+import defaultTemplate from './quanticQuickviewDefault.html';
+// @ts-ignore
+import youtubeTemplate from './quanticQuickviewYoutube.html';
+
+
+/** @typedef {import("coveo").Result} Result */
+
+const allowedHost = {youtube: 'https://youtube.com/'};
+
+/**
+ * The `QuanticQuickviewContent` component renders a content template based on the type of the content (Youtube, other).
+ * @category Result Template
+ * @example
+ * <c-quantic-quickview-content result={result} content-url={contentURL}></c-quantic-quickview-content>
+ * @internal
+ */
+export default class QuanticQuickviewContent extends LightningElement {
+  /**
+   * The [result item](https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#result).
+   * @api
+   * @type {Result}
+   */
+  @api result;
+  /**
+   * The `src` path used to render the iframe when content type is not youtube.
+   * @api
+   * @type {String}
+   */
+  @api contentUrl;
+
+  /** @type {Boolean} */
+  isLoading = true;
+
+  handleIframeLoaded() {
+    this.isLoading = false;
+    this.dispatchEvent(new CustomEvent('loadingstatechange'));
+  }
+
+  stopPropagation(evt) {
+    evt.stopPropagation();
+  }
+
+  render() {
+    if (this.result?.uniqueId.includes(allowedHost.youtube)) {
```

Comment:
Interesting, wouldn't have thought to use that as the check, but I think it works well :+1: